Hardware/Computer.cs

```diff
@@ -208,6 +217,19 @@ public class Computer : IComputer {
       }
     }
 
+    public bool NICEnabled {
+      get { return nicEnabled; }
      [SecurityPermission(SecurityAction.LinkDemand, UnmanagedCode = true)]

```

Comment:
SP should be on other line.

Reply:
Line Ending error -> Please convert after merge